src/init.cpp

```diff
@@ -1394,7 +1375,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                 }
 
                 // Check for changed -insightexplorer state
-                if (fInsightExplorer != GetBoolArg("-insightexplorer", false)) {
+                if (fExperimentalInsightExplorer != fInsightExplorer) {
```

Comment:
Minor nit, but I think the original ordering is more intuitive (i.e. ``fInsightExplorer != fExperimentalInsightExplorer``).